sparse_ho/algo/implicit.py

```diff
@@ -39,20 +71,49 @@ def get_beta_jac_v(
 
 def get_beta_jac_t_v_implicit(
         X, y, log_alpha, get_v, mask0=None, dense0=None, tol=1e-3,
-        model="lasso", sk=False, max_iter=1000, sol_lin_sys=None):
-    alpha = np.exp(log_alpha)
-    n_samples, n_features = X.shape
+        model="lasso", max_iter=1000, sol_lin_sys=None,
+        tol_lin_sys=1e-6, max_iter_lin_sys=100):
+    """Algorithm that will compute the (hyper)gradient, ie the gradient with
+    respect to the hyperparameter using the implicit differentiation.
 
-    mask, dense, _ = get_beta_jac_iterdiff(
+    Parameters
+    ----------
+    X: array-like, shape (n_samples, n_features)
+        Design matrix.
+    y: ndarray, shape (n_samples,)
+        observation vector.
+    log_alpha: float or np.array, shape (n_features,)
+        logarithm of hyperparameter.
+    mask0: ndarray, shape (n_features,)
+        boolean of active active for warm start
+    dense0: ndarray, shape (n_features,)
+        initial value of the value on the support for warm start
+    tol: float
+        the tolerance for the optimization
+    model:  instance of ``sparse_ho.base.BaseModel``
+        a model that follows the sparse_ho API.
+    max_iter: int
+        maximum number of iteration for the inner solver
+    sol_lin_sys: ndarray
+        previous solution of the linear system for warm start
```

Comment:
which linear system? At the top of the docstring, say we first find hat beta, then solve a linear system involving it and its generalized support

Reply:
I added this int the doc string:
`First the algorithm computes the regression coefficients beta, then the gradient is computed after resolution of a linear system on the generalized  support of the regression coefficients.`